modules/rosa-hcp/vars.tf

```diff
@@ -25,7 +25,7 @@ variable "replicas" {
 variable "compute_node_instance_type" {
   type        = string
   description = "The EC2 instance type to use for compute nodes"
-  default     = "m5.xlarge"
+  default     = "m7.xlarge"
```

Comment:
afaik m7 does not exist.
E.g. and would go with e.g. `m7i.xlarge`.

m5 was the last one that just had m5 and starting with m6 they started splitting it up further.